poc/vdaf_prio3.py

```diff
@@ -483,10 +483,34 @@ class Prio3HistogramWithLength(Prio3Histogram):
         return Prio3HistogramWithLength
 
 
+class Prio3MultiHotHistogram(Prio3):
+    # Generic types required by `Prio3`
+    Xof = xof.XofShake128
+
+    # Associated parameters.
+    VERIFY_KEY_SIZE = xof.XofShake128.SEED_SIZE
+    ID = 0x00000FFF  # private codepoint for Prio3 types
```

Comment:
This codepoint space is intended for Prio3 codepoints that are standardized. Let's leave it out until we've decided to do so. (If it turns out that we do want this for the DP draft, and the DP draft gets adopted, then we will certainly add it to the spec!)
```suggestion
    ID = 0xFFFFFFFF  # private codepoint for Prio3 types
```